lib/evmone/instructions.cpp

```diff
@@ -1377,6 +1377,11 @@ constexpr op_table create_op_table_constantinople() noexcept
 constexpr op_table create_op_table_istanbul() noexcept
 {
     auto table = create_op_table_constantinople();
+    table[OP_BALANCE] = {op_balance, 700, 1, 0};
+    table[OP_CHAINID] = {op_undefined, 2, 0, 1};
+    table[OP_EXTCODEHASH] = {op_extcodehash, 700, 1, 0};
+    table[OP_SELFBALANCE] = {op_undefined, 5, 0, 1};
```

Comment:
Shouldn't this and chainid be implemented?

Reply:
They are in separate PRs, it's better to merge this first, and then rebase those